README.md

```diff
@@ -70,9 +70,9 @@ You can ask and answer questions, have open-ended conversations, and follow alon
 
 ✉️ [Mail us](mailto:info@penpot.app)
 
-💬 [Github discussions](https://github.com/penpot/penpot/discussions)
+💬 [GitHub discussions](https://github.com/penpot/penpot/discussions)
 
-🐞 [Github issues](mailto:info@penpot.apphttps://github.com/penpot/penpot/issues)
+🐞 [GitHub issues](https://github.com/penpot/penpot/issues)
```

Comment:
This broken link is user visible. (the spell checker didn't like `apphttps` but since I've made a significant change to the url, I've given the commit a distinctive commit message).